### PR TITLE
[Docs] Document redirect parameters

### PR DESCRIPTION
**Pull request by delbaoliveira**
Fixes: 

- https://github.com/vercel/next.js/issues/51975
- [Linear](https://linear.app/vercel/issue/DX-1713/docs-redirect-is-missing-an-explanation-not-redirecttype)

### CODE REVIEW

File: docs/02-app/02-api-reference/04-functions/redirect.mdx [hunk 1]
```diff
@@ -3,11 +3,32 @@ title: redirect
...
+```
+redirect(path, type)
+```
+
+| Parameter | Type                    | Default     | Description                                                 |
+| --------- | ----------------------- | ----------- | ----------------------------------------------------------- |
+| `path`    | `string`                | -           | The URL to redirect to. Can be a relative or absolute path. |
+| `type`    | `'replace'` or `'push'` | `'replace'` | The type of redirect to perform.                            |
+
+By default, `redirect` will replace the current URL in the browser history stack. If you want to add a new entry to the history stack (e.g. when using it in a server action to allow the user to return to the previous page), use `push` instead.
```
Comment (feedthejim) [inline docs/02-app/02-api-reference/04-functions/redirect.mdx:23]: server actions redirect calls do this by default

Comment (delbaoliveira) [reply docs/02-app/02-api-reference/04-functions/redirect.mdx:23]: Sorry, do what by default?

Comment (alvarlagerlof) [inline docs/02-app/02-api-reference/04-functions/redirect.mdx:23]: Might be the same thing, but what about calling redirect in server components?

**Review by feedthejim — APPROVED** [review]

File: docs/02-app/02-api-reference/04-functions/redirect.mdx (hunk 1 above)
Comment (delbaoliveira) [reply docs/02-app/02-api-reference/04-functions/redirect.mdx]: @alvarlagerlof update the paragraph, hopefully that's clearer! 🤞🏼

**Pull request merged**